releasenotes-installscript/notes/rpm-gpgkeys-improvements-3456dc45d103b818.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG-INSTALLSCRIPT.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    * Add a `gpgkey=` entry pointing to a keyfile that always contains key used to signed current RPM repodata.
```

Comment:
I don't have a good suggestion for rewording but I think this would be better if rephrased more in "end user" terms (since one does not need to know what the `gpgkey` entry is to use this)

Reply:
Good point, I'll try to figure out something better 👍

Reply:
Pushed. How about now? :)